client/src/components/Snackbar/index.jsx

```diff
@@ -85,19 +98,28 @@ MySnackbarContentWrapper.propTypes = {
   variant: PropTypes.oneOf(['error', 'info', 'success', 'warning']).isRequired,
 };
 
-export default function CustomizedSnackbars({ message, variant, timeopened }) {
-  const [open, setOpen] = React.useState(false);
+const CustomizedSnackbars = ({ message, variant, timeopened }) => {
+  // const [open, setOpen] = React.useState(false);
+  const dispatch = useDispatch();
+
+  const { open } = useSelector((state) => state.snackbarReducer);
 
   useEffect(() => {
-    setOpen(true);
-  }, [timeopened]);
+    const payload = {
+      open: true,
+    };
+    dispatch({ type: SET_SNACKBAR_OPEN, payload });
+  }, [timeopened, dispatch]);
```

Comment:
I'm not sure why this is in a `useEffect`

Reply:
Every time a new unix time `timeopened` is passed as a prop, the `SET_SNACKBAR_OPEN` is dispatched. Just a way to handle triggering this action.